rero_ils/modules/items/api.py

```diff
@@ -547,10 +547,35 @@ def last_location_pid(self):
             return loan_location_pid
         return self.location_pid
 
+    def can_extend(self, loan):
+        """Checks if an item can be renewed for the borrowed patron."""
+        from ..circ_policies.api import CircPolicy
```

Comment:
Is this can be placed at the top of the file?

Reply:
Done